### PR TITLE
feat: added mapzy logo and switched to importmaps

### PR DESCRIPTION
**Pull request by shafy**
Added the Mapzy logo and moved the mapbox logo to the top left. I will wait for Dorena to make a black and white version of the Mapzy logo to match the Mapbox logo before merging (see [Basecamp](https://3.basecamp.com/4514167/buckets/21367310/uploads/4434773908)).

Screenshot 2022-02-06 at 15 19 45

Do you think the positioning like this is good? Later on we can also add a setting that let's you hide the Mapzy logo, but didn't do it yet.

Also moved away from webpacker to importmaps. Updated Foreman and Gitpod setup accordingly.

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -1,71 +0,0 @@
...
-      ],
-      [
-        '@babel/plugin-transform-regenerator',
-        {
-          async: false
-        }
-      ]
-    ].filter(Boolean)
-  }
-}
```
Comment (sagzy) [inline babel.config.js:71]: :D

**Review by sagzy — COMMENTED** [review]
Yay! Love the cleanup & simplification of the JS setup. :D 

Added 2 tiny comments on the code + there is one error blocking the CI

File: Procfile.dev [hunk 2]
```diff
@@ -1,3 +1,3 @@
 web: bundle exec rails s -p 3000
-webpack: ./bin/webpack-dev-server
-sidekiq: bundle exec sidekiq -q default -q mailers
\ No newline at end of file
+tailwind: rails tailwindcss:watch
+sidekiq: bundle exec sidekiq -q default -q mailerscss: bin/rails tailwindcss:watch
```
[the remark this answers is not in the document]
Comment (shafy) [reply Procfile.dev:3]: Ahh, I was thinking wtf was that. It looks like the tailwind installer added this automatically but didn't add a newline. Amateuer hour over there, DHH.

**Comment by shafy** [comment]
@sagzy can you already review this? I want to change the logo to a black and white one later but the rest works and it also fixes the bug we have in production where you can't delete a location. I will add the new logo in a separate PR.

**Comment by shafy** [comment]
I guess I also need to update the `Dockerfile` since we don't need Node anymore...

File: app/views/dashboard/accounts/settings.html.erb [hunk 3]
```diff
@@ -15,7 +15,10 @@
         <p class="mb-4"><%= current_user.email %></p>
 
         <%= link_to "Change Email or Password", edit_user_registration_path %> <br>
-        <%= link_to "Log out", destroy_user_session_path, method: :delete %>
+        <%= button_to "Log out", destroy_user_session_path,
+            method: :delete,
+            class: "no-underline cursor-pointer text-mapzy-red hover:text-mapzy-red-light font-semibold"
+        %>
```
Comment (sagzy) [inline app/views/dashboard/accounts/settings.html.erb:21]: Weird that it doesn't work with `link_to` and `method: :delete`, even though it's written in the official [documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to). I also kind of remember trying this out and it worked. Maybe a browser thing?

Comment (sagzy) [reply app/views/dashboard/accounts/settings.html.erb:21]: thanks anyways for fixing it

File: app/views/dashboard/locations/_form.html.erb [hunk 4]
```diff
@@ -98,12 +98,12 @@
...
                       <%= fot.hidden_field :day %>
 
-                      <div class="py-1 my-1 mr-8 w-full xl:w-1/5">
+                      <div class="py-1 my-1 mr-6 w-full xl:w-1/5">
                         <span class="uppercase tracking-wide text-gray-700 text-xs font-bold"><%= opening_time.day %></span>
                       </div>
-
-                      <div class="flex flex-row justify-start md:justify-between items-start py-1 mr-4 text-sm"
-                           data-opening-time-target="timesBlock" >
+
```
Comment (sagzy) [inline app/views/dashboard/locations/_form.html.erb:104]: trailing spaces

**Review by sagzy — APPROVED** [review]
Great great great. thanks! Let's merge it

(1 lil' comment about trailing spaces)

File: app/views/dashboard/accounts/settings.html.erb (hunk 3 above)
Comment (shafy) [reply app/views/dashboard/accounts/settings.html.erb:21]: It's a Rails 7 + Turbo thing. Turbo is annoying the shit out of me, haha.

**Pull request merged**